tests/tests_geomstats/test_hyperbolic.py

```diff
@@ -559,3 +559,15 @@ def test_parallel_transport_vectorization(self):
         results = helper.test_parallel_transport(space, metric, shape)
         for res in results:
             self.assertTrue(res)
+
+    def test_projection_and_belongs(self):
```

Comment:
Not necessarily for this PR, but the unit tests could be refactored too. For example, we could have a TestHyperbolic inherit from TestEmbeddedManifold, and directly run a `test_projection_and_belongs` function on the hyperbolic space. Same for the other embedded manifolds.